### PR TITLE
Update Windows section of README

### PR DESCRIPTION
**Pull request by dra27**

### CODE REVIEW

**Review by rjbou — APPROVED** [review]

**Comment by kit-ty-kate** [comment]
The [blog post](https://opam.ocaml.org/blog/opam-2-2-0-alpha2/) also specifies `autoconf` (I’m not entirely sure why) and `curl` (required for `make cold` and at runtime)

File: README.md [hunk 1]
```diff
@@ -51,26 +51,20 @@ If you are developing opam, you may enable developer features by including the
...
                         pre-installed)
-* From Devel - `mingw64-i686-gcc-core` & `mingw64-x86_64-gcc-core` (not required if
+* From Devel - `mingw64-i686-gcc-g++` & `mingw64-x86_64-gcc-g++` (not required if
                                                                  building with MSVC)
 
 Alternatively, having downloaded Cygwin's setup program, Cygwin can be installed
 using the following command line:
 
-`setup-x86_64 --root=C:\cygwin64 --quiet-mode --no-desktop --no-startmenu --packages=make,mingw64-i686-gcc-core,mingw64-x86_64-gcc-core,patch`
+`setup-x86_64 --root=C:\cygwin64 --quiet-mode --no-desktop --no-startmenu --packages=make,mingw64-i686-gcc-g++,mingw64-x86_64-gcc-g++,patch`
```
Comment (rjbou) [inline README.md:67]:
```suggestion
`setup-x86_64 --root=C:\cygwin64 --quiet-mode --no-desktop --no-startmenu --packages=make,mingw64-i686-gcc-g++,mingw64-x86_64-gcc-g++,patch,autoconf,curl`
```

File: README.md [hunk 2]
```diff
@@ -51,26 +51,20 @@ If you are developing opam, you may enable developer features by including the
...
-32-bit opam using 64-bit Cygwin and vice versa though. Note that you must be running
-64-bit Windows in order to build the 64-bit version).
+Windows.
 
 The following Cygwin packages are required:
 * From Devel - `make`
 * From Devel - `patch` (not required if OCaml and all required packages are
                         pre-installed)
-* From Devel - `mingw64-i686-gcc-core` & `mingw64-x86_64-gcc-core` (not required if
+* From Devel - `mingw64-i686-gcc-g++` & `mingw64-x86_64-gcc-g++` (not required if
```
Comment (rjbou) [inline README.md:61]:
```suggestion
* From Devel - `autoconf`
* From Devel - `curl`
* From Devel - `mingw64-i686-gcc-g++` & `mingw64-x86_64-gcc-g++` (not required if
```

**Review by rjbou — APPROVED** [review]

**Comment by rjbou** [comment]
@kit-ty-kate updated according blog post

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**